gpt_2_simple/gpt_2.py

```diff
@@ -573,32 +570,32 @@ def copy_checkpoint_to_gdrive(run_name='run1', copy_folder=False):
     """Copies the checkpoint folder to a mounted Google Drive."""
     is_mounted()
 
-    checkpoint_folder = os.path.join('checkpoint', run_name)
+    checkpoint_folder = f'checkpoint/{run_name}'
```

Comment:
More efficient, and `os.path.join()` was unneeded, since this is a command that can only be run on Colaboratory with the `/content/XYZ` references later in the function.